### ISSUE TITLE
Introduce Option to create Applications in an Assembly concurrently

### ISSUE THREAD

**Issue by brianoliver** [coherence-community/oracle-bedrock#326]
As a developer, often I'd like to start Applications in parallel.  To achieve this an AssemblyBuilder should accept an option when building an assembly to specify if building should occur concurrently, and perhaps, how many at a time!

**Comment by brianoliver** [comment]
Won't Fix as Applications are always launched asynchronously.
